index.bs

```diff
@@ -170,19 +182,68 @@ When an image item is of type <dfn value export for="AV1 Image Item Type">av01</
 The syntax and semantics of the <dfn>AV1 Item Configuration Property</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
 
   - [=Sequence Header OBUs=] should not be present in the [=AV1CodecConfigurationBox=].
-  - If a [=Sequence Header OBU=] is present in the [=AV1CodecConfigurationBox=], it shall match the  [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
+  - If a [=Sequence Header OBU=] is present in the [=AV1CodecConfigurationBox=], it shall match the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
   - The values of the fields in the [=AV1CodecConfigurationBox=] shall match those of the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
   - [=Metadata OBUs=], if present, shall match the values given in other item properties, such as the PixelInformationProperty or ColourInformationBox.
 
 This property shall be marked as essential.
 
-<h4 id ="other-item-property">Other Item Properties</H4>
+<h4 id ="other-item-property">Other Item Properties</h4>
 
 In addition to the Image Properties defined in [[!HEIF]], such as 'colr', 'pixi' or 'pasp', [=AV1 image items=] MAY also be associated with 'clli', 'cclv' and 'mdcv' introduced in [[!MIAF]].
 
 In general, it is recommended to use properties instead of [=Metadata OBUs=] in the [=AV1 Item Configuration Property=].
 
-For multi-layered images, the LayerSelectorProperty may be present as defined in [[!HEIF]].
+<h3 id ="layered-items">AV1 Multi-Layered Image Items</h3>
+
+<h4 id ="layered-items-overview">Overview</h4>
+When multi-layer encoding is used as defined in [[!AV1]] to encode an [=AV1 Image Item=], the item is called an <dfn>AV1 Multi-Layered Image Item</dfn>. Its [=AV1 Image Item Data=] contains multiple [=AV1 Frames=], each corresponding to a different spatial layer, and shall conform to specific constraints given in [[!AV1]], for example on the use of [=OBU extension headers=] or on the ordering of [=OBUs=].
+
+In an [=AV1 Multi-Layered Image Item=], a spatial layer may improve the resolution or quality of the image decoded based on one or more of the previous layers. A layer may also provide an image that does not depend on the previous layers. Not all layers are expected to produce an image meant to be rendered. Some decoded images may be used only as intermediate encodes. The [=Sequence Header OBU=] defines the combinations of layers that are expected to produce output images. Each combination is called an [=Operating Point=]. An [=AV1 Image Item=] may contain data from multiple [=Operating Points=]. It may have [=Operating Points=] representing the same image in multiple resolutions or qualities. It may have [=Operating Points=] representing different images encoded based on a common set of layers.
```

Comment:
Cyril: In the sentence

> Some decoded images may be used only as intermediate encodes.

did you mean "intermediate encodes" or "intermediate decodes"?

Reply:
The sentence is mixing both and is not correct, but depending on the point of view, the intermediate images can be viewed as intermediate encodes (from an encoder's point of view) or intermediate decodes (from a decoder's point of view).

Reply:
Fixed